### PR TITLE
Added urdf & mesh files for D405 model

### PR DESCRIPTION
**Pull request by Arun-Prasad-V**
Tracked on RSDSO-18600

Related issue: https://github.com/IntelRealSense/realsense-ros/issues/2589

### CODE REVIEW

File: realsense2_description/urdf/_d405.urdf.xacro [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+        <material name="aluminum"/>
+      </visual>
+      <collision>
+        <origin xyz="${d405_zero_depth_to_glass-d405_cam_depth/2} ${-d405_cam_depth_py} 0" rpy="0 0 0"/>
+        <geometry>
+          <box size="${d405_cam_depth} ${d405_cam_width} ${d405_cam_height}"/>
+        </geometry>
+      </collision>
+      <inertial>
+        <!-- The following are not reliable values, and should not be used for modeling -->
```
Comment (Arun-Prasad-V) [inline realsense2_description/urdf/_d405.urdf.xacro:89]: The <inertial> section hasn't been updated. It has unreliable values. 
In fact, all the previous URDF's of D415, D455, D435 models also have same unreliable values.

File: realsense2_description/urdf/_d405.urdf.xacro [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+    <!-- The following values are approximate, and the camera node
+     publishing TF values with actual calibrated camera extrinsic values -->
+    <xacro:property name="d405_cam_depth_to_infra1_offset" value="0.0"/>
+    <xacro:property name="d405_cam_depth_to_infra2_offset" value="-0.018"/>
+    <xacro:property name="d405_cam_depth_to_color_offset" value="0.0"/>
+
+    <!-- The following values model the aluminum peripherial case for the
+  	d405 camera, with the camera joint represented by the actual
+  	peripherial camera tripod mount -->
+    <xacro:property name="d405_cam_width" value="0.042"/>
```
Comment (Nir-Az) [inline realsense2_description/urdf/_d405.urdf.xacro:42]: Just curios , this is generated automatically some how or you set this values according to the STL file manually?

**Comment by Nir-Az** [comment]
Can you attach a visualization of the output?
*Closed and reopened by mistake :)

File: realsense2_description/urdf/_d405.urdf.xacro (hunk 2 above)
Comment (Arun-Prasad-V) [reply realsense2_description/urdf/_d405.urdf.xacro:42]: These values are set manually based on the dimensions mentioned in the datasheet.

**Comment by Arun-Prasad-V** [comment]
FYR...

https://github.com/IntelRealSense/realsense-ros/assets/127019120/bbfb6fdd-d715-4ff8-aee3-e4c48c79f165

**Comment by Nir-Az** [comment]
Please verify if we can expose the optical lens as discussed

**Comment by Arun-Prasad-V** [comment]
> Please verify if we can expose the optical lens as discussed

The STEP file itself doesn't have the details of lens, so couldn't generate the mesh file with lens exposed.

**Review by Nir-Az — APPROVED** [review]

**Pull request merged**